### PR TITLE
remove superseded non-LTS OpenJDK test jobs

### PR DESCRIPTION
**Pull request by richardstartin**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -263,7 +263,7 @@ workflows:
           name: test_<< matrix.testTask >>
           matrix:
             parameters:
-              testTask: [ "7", "IBM8", "ZULU8", "11", "ZULU11", "13", "ZULU13", "14", "15" ]
+              testTask: [ "7", "IBM8", "ZULU8", "11", "ZULU11", "ZULU13", "15" ]
```
Comment (bantonsson) [inline .circleci/config.yml:266]: So does `ZULU13` get special treatment because of Azul MTS, or should we switch to `ZULU15`?

Comment (richardstartin) [reply .circleci/config.yml:266]: We should move to ZULU15 but we need to update the docker image first I believe 

https://github.com/DataDog/dd-trace-java-docker-build/blob/master/Dockerfile

Comment (richardstartin) [reply .circleci/config.yml:266]: in fact it seems to acknowledge zulu15

**Review by bantonsson — APPROVED** [review]
Too bad this can never be merged since the required steps `test_13` and `test_14` will never finish 😉

**Review by mcculls — APPROVED** [review]

**Pull request merged**